platform/darwin/src/NSDictionary+MGLAdditions.mm

```diff
@@ -0,0 +1,51 @@
+#import "NSDictionary+MGLAdditions.h"
+
+#import "NSExpression+MGLAdditions.mm"
+
+@implementation NSDictionary (MGLAdditions)
+
+- (mbgl::PropertyMap)mbgl_propertyMap {
+    mbgl::PropertyMap propertyMap;
+    return [self propertyMapWithPropertyMap:propertyMap dictionary:self];
+}
+
+- (mbgl::PropertyMap)propertyMapWithPropertyMap:(mbgl::PropertyMap &)propertyMap dictionary:(NSDictionary *)dictionary {
```

Comment:
This was fixed in https://github.com/mapbox/mapbox-gl-native/pull/6524/commits/bc1dda36f6fabb272f273749b8b7cd6121608c48. Note that`propertyMapWithDictionary:` and `vectorForArray:` (name change in https://github.com/mapbox/mapbox-gl-native/pull/6524/commits/95d548b9f2e437e9a3eae2b10860bd0b39fbef04) still take `NSDictionary`s and `NSArray`s so that the entire dictionary can be traversed (recursively) including if any node is a dictionary or array. The `(mbgl::PropertyMap)mbgl_propertyMap` method already exists and is the only public method of the category. It is intended to take no parameters and return a property map.